state/kvstore.go

```diff
@@ -32,107 +73,80 @@ func (mkv *MemKVStore) Get(key []byte) (value []byte) {
 	return mkv.m[string(key)]
 }
 
-//----------------------------------------
-
-// A Cache that enforces deterministic sync order.
-type KVCache struct {
-	store    KVStore
-	cache    map[string]kvCacheValue
-	keys     *list.List
-	logging  bool
-	logLines []string
+func (mkv *MemKVStore) Has(key []byte) (has bool) {
```

Comment:
Just a random question: whenever I see a key-value store immediately I imagine concurrent usage, would that apply for this one?